terraform/Makefile

```diff
@@ -4,7 +4,7 @@
 #ENVIRONMENT = gx-betacloud
 #ENVIRONMENT ?= gx-citycloud
 #ENVIRONMENT = gx-scs
-SHELL=/bin/bash
+SHELL = /bin/bash
```

Comment:
IMO it is not a good practice to address formatting issues in a pull request intended for a bug fix. I personally prefer having a separate pull request dedicated specifically to formatting changes.